configuring/configuring-openshift-builds.adoc

```diff
@@ -7,9 +7,13 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 
-In a `Build` custom resource (CR), you can define the source, build strategy, parameter values, output, retention parameters, and volumes to configure a build. A `Build` resource is available for use within a namespace.
+In a `Build` custom resource (CR), you can define the source, build strategy,
```

Comment:
```suggestion
In a `Build` custom resource (CR), you can configure a build by defining the source, build strategy, parameter values, output, retention parameters, and volumes. You can also configure build pods by using a `Build` CR.
```
Similar edits here as what I suggested in `configuring-build-runs.adoc`. See my comments there.